production.env

```diff
@@ -0,0 +1,21 @@
+SECRET_KEY_BASE=""
+PAPERCLIP_HASH_SECRET=""
+
+DATABASE_URL=postgresql://postgres@db/scinote_production
+
```

Comment:
So, if user wants these changed for production, he/she just changes this file? Should that really be in Git repository?